server/src/compression.rs

```diff
@@ -445,11 +487,16 @@ mod tests {
 
     #[test]
     fn test_try_compress_regular_almost_constant_time_series() {
-        let timestamps = Vec::from_iter((100..1000).step_by(100));
-        let values = vec![10.1, 10.0, 10.2, 10.2, 10.0, 10.1, 10.0, 10.0, 10.0];
+        let values = comp_test_util::generate_values(
+            TRY_COMPRESS_TEST_LENGTH,
+            comp_test_util::DataType::Random,
```

Comment:
Maybe add a comment why `DataType::Random` is used to generate almost constant values.

Reply:
Fixed.